app/scripts/modules/core/src/ci/igor.service.ts

```diff
@@ -58,7 +58,7 @@ export class IgorService {
       .get();
   }
 
-  public static getGcbAccounts(): IPromise<String[]> {
+  public static getGcbAccounts(): IPromise<string[]> {
```

Comment:
:see_no_evil: This was definitely me...thank you so much for doing this! :pray: :nail_care: